code/modules/mob/living/silicon/pai/pai.dm

```diff
@@ -553,12 +553,15 @@
 
 /mob/living/silicon/pai/MouseDrop(atom/over_object)
 	var/mob/living/carbon/H = over_object
-	if(!istype(H) || !Adjacent(H)) return ..()
-	if(H.a_intent == I_HELP)
-		get_scooped(H)
-		//return
-	else
-		return ..()
+	if(!istype(H) || !Adjacent(H) || usr != src) return ..()

```

Comment:
This will prevent anyone that is not the pAI from picking it up, a better solution would [be something like this](http://hastebin.com/sicelebupe.coffee)